endpoint simplifications

evgb opened the pull request.

AymericaRST committed:
Location endpoint

AymericaRST committed:
Media endpoint

AymericaRST committed:
Tag endpoint

AymericaRST committed:
User endpoint

AymericaRST committed:
some corrections

evgb committed:
Merge remote-tracking branch 'origin/Instagram-constructor-request-options'

evgb committed:
Merge branch 'master' into master

marvinosswald commented:
This does basically just remove the whole object capsulation around guzzle and just returns that directly, correct ?

evgb commented:
Yes, correct, i think it's quite easier to use without the encapsulation

kcassam commented:
With config ['http_errors' => false], you sometimes receive a response that is not convertible into the awaited object but is an error message. See for example : https://github.com/marvinosswald/instagram-php/commit/b83464dca650701bb59eed5f8ad2fdb66ab0d17c#diff-b946710c6396824029216b7cc1a60a3bR47

evgb committed:
rolled back tests for simplified endpoints

also added two environment variables for testing with differents access tokens in UserTest

evgb committed:
Merge branch 'master' of https://github.com/evgb/instagram-php

evgb committed:
fixes

removed useless INSTAGRAM_SEARCH
and INSTAGRAM_USERNAME usages

evgb committed:
added failure tests for every endpoint
